api_pagopa.yaml

```diff
@@ -39,9 +39,9 @@ paths:
             $ref: "#/definitions/ProblemJson"
 definitions:
   EmailAddress:
-    $ref: "https://raw.githubusercontent.com/teamdigitale/io-functions-commons/v1.4.0/openapi/definitions.yaml#/EmailAddress"
+    $ref: "https://raw.githubusercontent.com/teamdigitale/io-functions-commons/v1.5.0/openapi/definitions.yaml#/EmailAddress"
```

Comment:
Do yo use some script to upgrade the api version ? (maybe we can put it in the scripts folder) I fear that at some point someone will forget to change one or some of these files upgrading :) 
cc  @fpersico

Reply:
No, I'll change the version manually every time. Sometimes it's not enough, some new definition must be added like [here](https://github.com/teamdigitale/io-backend/pull/536/files#diff-aa6cd0613984937586c55eb6ed53d7baR631)